guides/tools/soft-phone.md

```diff
@@ -0,0 +1,36 @@
+# Soft-phone
+
+Soft-phone allows you you to make a calls over the Internet using a general purpose computer rather than dedicated hardware. Mostly we use it to make calls to Twilio endpoints. There are lot of different clients, but this doc will explain how to configure Zoiper client.
```

Comment:
Soft-phones allow you you to make phonecalls over the Internet using your computer rather than dedicated hardware. Soft-phones are great for manual testing of Twilio apps. There are lot clients to choose from, but this doc will explain how to configure Zoiper.